sympy/simplify/cse_main.py

```diff
@@ -40,6 +40,197 @@
 # ===============================================================
 
 
+class CseExpr(tuple):
+    """Type of objects to be returned by the ``cse`` function.
+
+    ``CseExpr`` is backwards compatible with previous objects returned by the
```

Comment:
I don't like docs that talk about "the way things used to be". This sort of thing just becomes less and less relevant as time goes on. I would just say that this class acts like the following type.

Reply:
Addressed in https://github.com/sympy/sympy/pull/24649/commits/e173d4074e40876a99219d10ca7bdf0e0d0d0bd0.